src/Connections/GithubRepo.ts

```diff
@@ -107,6 +87,119 @@ const AllowedEvents: AllowedEventsNames[] = [
     "release",
 ];
 
+const ConnectionStateSchema = {
+  type: "object",
+  properties: {
+    priority: {
+        type: "number",
+        nullable: true,
```

Comment:
This would only compile after peppering the schema will these `nullables`. I suppose it's correct, but it's also naff.